.github/workflows/test.yml

```diff
@@ -46,21 +48,52 @@ jobs:
   build-docker:
     runs-on: ubuntu-latest
     container:
-      image: ubuntu:focal
+      image: osrf/ros2:nightly
     steps:
-      - name: Setup ROS2
-        uses: ros-tooling/setup-ros@2c05014
+      - name: prep
+        run: |
+          apt-get -qq update
+          apt-get -qq upgrade -y
+          apt-get -qq install -y curl libasio-dev libtinyxml2-dev
+          colcon mixin add default https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
+          colcon mixin update default
+          mkdir -p src
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: src/
+
+      - run: cp src/codecov.yml .
+
+      - name: Rosdep
+        run: DEBIAN_FRONTEND=noninteractive rosdep install -r --from-paths src --ignore-src --rosdistro foxy -y || true
```

Comment:
This was one of my main complaints from the tooling WG's actions-- this always succeeds (both with the -r and the || true). Wouldn't we prefer failure here?

Reply:
True. I'll change that

Reply:
95ecb6b

Reply:
This is still not fixed. The above commit only changed line 58.

Reply:
I meant 6b964feaec6111a19e03933b1f9751aa6e210ed9 but forgot to push 👀

Reply:
THERE we go, thank you.